### PR TITLE
Cleanup and add kafka

### PR DESCRIPTION
**Pull request by steinbrueckri**
## Changes:
- Remove ZSH and fish because nobody is using them.
- Add kafkactl to debug Kafka connections

### CODE REVIEW

**Comment by steinbrueckri** [comment]
@gleichda do you want to take a look at it, yes I know it's not totally sexy but I had no better idea at first.

File: src/packages.txt [hunk 1]
```diff
@@ -11,7 +11,6 @@ tree
 vim
 curl
 openssh-server
-zsh
```
Comment (gleichda) [inline src/packages.txt:14]: 😢

**Review by gleichda — APPROVED** [review]

File: src/packages.txt (hunk 1 above)
Comment (steinbrueckri) [reply src/packages.txt:14]: If you need it I will drop the commit ...

Comment (gleichda) [reply src/packages.txt:14]: No we definitely need to reduce the size... So good for me

**Review by gleichda — APPROVED** [review]

**Pull request merged**